examples/MySQL-client/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ project(MySQL-client-example)
 hunter_add_package(MySQL-client)
 find_package(MySQL-client REQUIRED)
 
+if(MSVC) #LIBCMT conflicts with MSVCR
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:LIBCMT.lib")
```

Comment:
> Am I doing something more than needs to be or is that just the way it is?

If you want to test it yourself then it sounds like a solution, clone `pkg.{name}` and merge it with your updates.